pyriemann/clustering.py

```diff
@@ -37,20 +37,17 @@ def _init_centroids(X, n_clusters, init, random_state, x_squared_norms):
 #######################################################################
 
 
-def _fit_single(X, y=None, n_clusters=2, init='random', random_state=None,
-                metric='riemann', max_iter=100, tol=1e-4, n_jobs=1):
+def _fit_single(X, y=None, n_clusters=2, init="random", random_state=None,
+                metric="riemann", max_iter=100, tol=1e-4, n_jobs=1):
     """helper to fit a single run of centroid."""
     # init random state if provided
     mdm = MDM(metric=metric, n_jobs=n_jobs)
     mdm.metric_mean, mdm.metric_dist = check_metric(metric)
-    squared_norms = [np.linalg.norm(x, ord='fro')**2 for x in X]
+    squared_norms = [np.linalg.norm(x, ord="fro")**2 for x in X]
```

Comment:
you could use: `np.linalg.norm(A, ord="fro", axis=(1, 2))**2`

Reply:
Yes, good idea!